### PR TITLE
Removes cash plants because I powergame with them to hard

### PR DESCRIPTION
**Pull request by ma44**

### CODE REVIEW

**Comment by kevinz000** [comment]
Local powergamer nerfs self. Mental status questioned by peers.

**Comment by Shapsy** [comment]
Local man removes cute feature because he has delusions of grandeur and needs attention

**Comment by PKPenguin321** [comment]
yeah these really should be removed though, there's been i think two PRs already where an attempt to nerf cargo was made because they had too many points or something

**Comment by GunHog** [comment]
It was his plan all along. It is one of the most effective ways of removing things.

**Comment by ExcessiveUseOfCobblestone** [comment]
﻿ＩＭＰＲＯＶＥ
ＤＯＮ＇Ｔ
ＲＥＭＯＶＥ

File: code/modules/hydroponics/grown/citrus.dm [hunk 1]
```diff
@@ -77,47 +77,3 @@
...
-/obj/item/weapon/reagent_containers/food/snacks/grown/shell/moneyfruit
-	seed = /obj/item/seeds/cash
-	name = "Money Fruit"
-	desc = "Looks like a lemon with something bulging from the inside."
-	icon_state = "moneyfruit"
-	bitesize_mod = 2
-
-/obj/item/weapon/reagent_containers/food/snacks/grown/shell/moneyfruit/add_juice()
-	..()
-	switch(seed.potency)
```
Comment (Jalleo) [inline code/modules/hydroponics/grown/citrus.dm]: This is the entire issue here why not overhaul it so up to 50 is just a larger stack of spacecash. E.g. 2 for each potency. And then past 50 instead it becomes stacks of c10s that become larger stacks at each point? E.g. Starts at 10 credit then goes up with a extra 10 c in each potency level.

Comment (ExcessiveUseOfCobblestone) [inline code/modules/hydroponics/grown/citrus.dm]: So would you do 
`if (x-x)
trash = (/obj/item/stack/spacecash) * ..() `

will it work like that?

**Comment by ma44** [comment]
Cause its to memey for a powergamer to nerf self

File: code/modules/hydroponics/grown/citrus.dm (hunk 1 above)
Comment (Jalleo) [inline code/modules/hydroponics/grown/citrus.dm]: Something like that.

**Pull request closed without merging**